test/lib/pineInProcess.ts

```diff
@@ -0,0 +1,29 @@
+import { exit } from 'process';
```

Comment:
Can you use snake-case for filenames for consistency please, ie `pine-in-process.ts` for this one (and the equivalent for others)